src/install.unit.test.ts

```diff
@@ -42,7 +42,7 @@ describe("install procedure", () => {
     beforeEach(() => {
         matlabInstallSystemDependenciesMock = matlab.installSystemDependencies as jest.Mock;
         matlabGetReleaseInfoMock = matlab.getReleaseInfo as jest.Mock;
-        matlabMakeToolcacheDirMock = matlab.makeToolcacheDir as jest.Mock;
+        matlabMakeToolcacheDirMock = matlab.getToolcacheDir as jest.Mock;
```

Comment:
Variable name needs to change to reflect new function name

Reply:
fixed